zhaquirks/legrand/remote_switch.py

```diff
@@ -0,0 +1,218 @@
+"""Device handler for Legrand Remote switch."""
+# import logging
+from typing import Any, List, Optional, Union
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.general import (
+    Basic,
+    BinaryInput,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    PollControl,
+    PowerConfiguration,
+)
+from zigpy.zcl.clusters.manufacturer_specific import ManufacturerSpecificCluster
+
+from zhaquirks import Bus, PowerConfigurationCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.legrand import LEGRAND
+
+MANUFACTURER_SPECIFIC_CLUSTER_ID = 0xFC01  # decimal = 64513
+
+VOLTAGE_REPORTED = "voltage_reported"
+
+
+class LegrandRemoteSwitchCluster(CustomCluster, ManufacturerSpecificCluster):
+    """LegrandRemoteSwitchCluster."""
+
+    cluster_id = MANUFACTURER_SPECIFIC_CLUSTER_ID
+    name = "LegrandRemoteSwitchCluster"
+    ep_attribute = "legrand_cluster"
+
+
+class LegrandRemoteSwitchLevelControl(CustomCluster, LevelControl):
+    """LegrandRemoteSwitchLevelControl."""
+
+    def bind(self):
```

Comment:
We should devise a better way to unbind the coordinator. @puddly tagging for thoughts.